class-vip-support-cli.php

```diff
@@ -115,28 +66,19 @@ public function remove_support_user( $args, $assoc_args ) {
 
 		$user_email = $args[0];
 
-		// Let's find the user
-		$user = get_user_by( 'email', $user_email );
+		$success = remove_support_user( $user_email );
 
-		if ( false === $user ) {
-			\WP_CLI::warning( "No user exists with the email address {$user_email}, so they could not be deleted" );
-			return;
+		if ( is_wp_error( $success ) && $success['code'] === 'invalid-user' ) {
+			return \WP_CLI::warning( "No user exists with the email address {$user_email}, so they could not be deleted" );
```

Comment:
Can `remove_support_user` just return this error message? Would avoid the need for the conditionals to check to the code.